pkg/kubectl/cmd/util/factory.go

```diff
@@ -206,6 +206,7 @@ const (
 	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName                 = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName                    = "configmap/v1"
+	ClusterRoleBindingV1GeneratorName           = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
My main concern is when using `kubectl run` we advice users to use `--generator` flag to be sure this works consistently between versions. But I guess the current directions are to get rid of run and rather replace it with targeted `kubectl create` commands. That's why I guess I'm good with that.